src/web/statics.rs

```diff
@@ -49,9 +49,22 @@ fn serve_file(req: &Request, file: &str) -> IronResult<Response> {
     let contents = ctry!(req, fs::read(&path));
 
     // If we can detect the file's mime type, set it
-    let content_type = MimeGuess::from_path(&path)
-        .first()
-        .map(|mime| ContentType(mime.as_ref().parse().unwrap()));
+    // MimeGuess misses a lot of the file types we need, so there's a small wrapper
+    // around it
```

Comment:
Can we upstream this patch? I don't want to have to special case things if I can at all help it.

Reply:
I'm going to guess not since there's lots of open issues & prs for exactly this with no response. It shouldn't really be a huge deal since there's not very many exceptions